themes/custom/campaign_base/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "Campaign_base",
   "version": "1.0.0",
   "dependencies": {
-    "@comicrelief/pattern-lab": "^5.1.7",
+    "@comicrelief/pattern-lab": "git://github.com/comicrelief/pattern-lab.git#e183fd0",
```

Comment:
Can you merge it first, please 1071_wrap_link_teaser_article to get right version